components/automate-cli/pkg/reportgenerator/usagegenerator.go

```diff
@@ -0,0 +1,218 @@
+package usagegenerator
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/dnlo/struct2csv"
```

Comment:
you are adding a new package here. We already using  "github.com/gocarina/gocsv" in our application.
can we use this instead of struct2csv? 
BTW, the Golang standard library has encoding/csv package to work with csv files.

Reply:
I used it since it was easier to convert struct to csv. But since we do not have a very complex structure gocsv can be used. encoding/csv would have required a lot more work to convert struct to string